app/views/admin/creator_settings/_form.html.erb

```diff
@@ -4,7 +4,7 @@
     <span class="crayons-field__required crayons-tooltip" data-tooltip="This will set the primary name for your Forem" aria-describedby="community-name-subtitle"></span>
     <p id="community-name-subtitle" class="crayons-field__description">Used as the primary name for your Forem.</p>
   <% end %>
-  <%= text_field_tag :community_name, "", placeholder: "Climbing Life", class: "crayons-textfield", required: true %>
+  <%= text_field_tag :community_name, "", placeholder: "Climbing Life", class: "crayons-textfield flex w-100 m:w-100", required: true %>
```

Comment:
Refactor (blocking): No need for any width CSS utility classes or the `flex` CSS utility class. The parent, `.crayons-field` has its display set to flex with a flex-flow of column.

![image](https://user-images.githubusercontent.com/833231/145596039-1460a740-8887-458f-bc55-52cf02fabe5a.png)

```suggestion
  <%= text_field_tag :community_name, "", placeholder: "Climbing Life", class: "crayons-textfield", required: true %>
```